glocaltokens/client.py

```diff
@@ -370,7 +371,7 @@ def get_google_devices(
             return devices
 
         network_devices: list[NetworkDevice] = []
-        if disable_discovery is False:
+        if address_list is None:
             LOGGER.debug("Getting network devices...")
```

Comment:
Can we please adjust the log message here? 
```suggestion
        if address_list is None:
            LOGGER.debug("Automatically discovering network devices...")
```